### PR TITLE
importer: Do not construct relative file:// URLs

### PR DESCRIPTION
**Pull request by camh-**
`file://` URLs cannot represent relative paths. The importer assumes
that they can by using a double-slash instead of a triple-slash. This is
wrong.

Force the path to be absolute when constructing a `file://` URL.

Drop coverage from 100 to 96.5 because it's really hard to make
`filepath.Abs()` return an error.

Closes: https://github.com/foxygoat/jsonnext/issues/12

### CODE REVIEW

File: importer.go [hunk 1]
```diff
@@ -122,7 +122,11 @@ func (i *Importer) readViaCache(imp string) (jsonnet.Contents, error) {
 		return content, nil
 	}
 
-	content, err := fetch(addScheme(imp), i.fetcher())
+	u, err := addScheme(imp)
+	if err != nil {
+		return noContent, err
+	}
+	content, err := fetch(u, i.fetcher())
```
Comment (juliaogris) [inline importer.go:129]: actually, not part of this PR, I think I'd invert the logic just below:
```go
content, err := fetch(u, i.fetcher())
if err != nil {
    return noContent, err
}
i.cache[imp] = content
return content, nil

Comment (camh-) [reply importer.go:129]: I think you're right. It did catch me out a little when i saw it again. Not sure why I did it this way.
